### ISSUE TITLE
Schema Tester Webpage not working

### ISSUE THREAD

**Issue by LukasAV** [hapijs/hapi.dev#363]
#### Support plan

* *which support plan is this issue covered by?* (e.g. Community, Core, Plus, or Enterprise): Community
* *is this issue currently blocking your project?* (yes/no): no
* *is this issue affecting a production system?* (yes/no): no

#### Context

* *node version*:  none
* *module version*:  latest
* *environment* (e.g. node, browser, native):  browser
* *used with* (e.g. hapi application, another framework, standalone, ...): nothing
* *any other relevant information*:  hapi.dev website

#### How can we help?

Hi, I really liked the online Schema Validator page at [https://hapi.dev/module/joi/tester/](https://hapi.dev/module/joi/tester/). However the page does not load lately. The errors displayed in the console are

```js
SyntaxError: invalid regexp group 2 da2434fe4ea5ef609506.js:1:1
ChunkLoadError: "Loading chunk 25 failed.
(missing: https://hapi.dev/_nuxt/da2434fe4ea5ef609506.js)"
    NuxtJS 55
ab3bd582bb5dd98bcefc.js:2:161133

```

I tried the page on multiple devices and browsers, but got always the same result. Is this error known?

**Comment by jarrodyellets** [comment]
Seems to be an issue with Firefox...it's working for me in Chrome.  I'll investigate.

**Comment by jarrodyellets** [comment]
Fixed with #364

**Comment by LukasAV** [comment]
Thank you very much!
